Jenkinsfile_CNP

```diff
@@ -334,46 +334,10 @@ def generateDefinitions() {
 
 def setFeatureFlags(environment) {
```

Comment:
Better to remove this section completely (move the rest of the flags to application.yaml)?